gtsfm/utils/io.py

```diff
@@ -153,6 +195,44 @@ def write_cameras(gtsfm_data: GtsfmData, images: List[Image], save_dir: str) ->
             f.write(f"{i} {camera_model} {image_width} {image_height} {fx} {u0} {v0} {k1} {k2}\n")
 
 
+def read_images_txt(fpath: str) -> Tuple[Optional[List[Pose3]], Optional[List[str]]]:
+    """Read camera poses and image file names from a COLMAP-format images.txt file.
+
+    Reference: https://colmap.github.io/format.html#images-txt
+        "The coordinates of the projection/camera center are given by -R^t * T, where
+        R^t is the inverse/transpose of the 3x3 rotation matrix composed from the
+        quaternion and T is the translation vector. The local camera coordinate system
+        of an image is defined in a way that the X axis points to the right, the Y axis
+        to the bottom, and the Z axis to the front as seen from the image."
+
+    Args:
+        fpath: path to images.txt file
+
+    Returns:
+        wTi_list: list of camera poses for each image, or None if file path invalid
+        img_fnames: name of image file, for each image, or None if file path invalid
```

Comment:
we have not been writing the variable names in the return section of the docstring.

Reply:
for a single return variable, i see Google python style doesn't list the variable name

but for many return variables, i think listing the name makes it easier to read

Reply:
I think we should follow a consistent style. :)